cmd/utils/flags.go

```diff
@@ -1100,6 +1100,13 @@ Please note that --` + MetricsHTTPFlag.Name + ` must be set to start the server.
 		Usage:    "Path for the voteJournal dir in fast finality feature (default = inside the datadir)",
 		Category: flags.FastFinalityCategory,
 	}
+
+	// Blob setting
+	BlobExtraReserveFlag = &cli.Uint64Flag{
+		Name:  "blob.extra-reserve",
+		Usage: "Extra reserve threshold for blob, blob never expires when 0 is set, default 28800",
+		Value: params.DefaultExtraReserveForBlobRequests,
+	}
```

Comment:
need Category
set to BlockHistoryCategory?

Reply:
Category for naming or something else?

Reply:
> Category for naming or something else?

the Category for the helper command to display